### PR TITLE
Guard hip header for COMGR PCH

### PR DESCRIPTION
**Pull request by JehandadKhan**

### CODE REVIEW

File: src/kernels/gpu_batched_transpose_kernel/batched_transpose.cpp [hunk 1]
```diff
@@ -23,8 +23,11 @@
  * SOFTWARE.
  *
  *******************************************************************************/
+#ifndef MIOPEN_DONT_USE_HIP_RUNTIME_HEADERS
 #include <hip/hip_runtime.h>
 #include <hip/hip_fp16.h>
+#endif
+
```
Comment (atamazov) [inline src/kernels/gpu_batched_transpose_kernel/batched_transpose.cpp:30]: Extra line

**Review by atamazov — APPROVED** [review]
LGTM

File: src/kernels/gpu_batched_transpose_kernel/batched_transpose.cpp (hunk 1 above)
Comment (JehandadKhan) [reply src/kernels/gpu_batched_transpose_kernel/batched_transpose.cpp:30]: Fixed

**Review by atamazov — APPROVED** [review]

**Review by carlushuang — APPROVED** [review]
LGTM

**Pull request merged**